webpages/popup/style.css

```
@@ -92,18 +90,31 @@ body {
   border-inline-end: none;
 }
 .popup-name.sel {
-  background-color: var(--brand-orange);
-  color: var(--white-text);
+  background-color: var(--hover-darken);
+  font-weight: bold;
+}
+.popup-name::before {
+  content: "";
+  display: block;
+  position: absolute;
+  left: 10px;
+  right: 10px;
+  bottom: 0;
+  height: 4px;
+  background-color: var(--orange);
```

I think this should be `--brand-orange` along with the category selector to keep consistency.
```suggestion
  background-color: var(--brand-orange);
```

---

I don't like the current approach of arbitrarily changing some elements to blue in prereleases but not others, but I don't think changing it belongs in this PR.